buf.gen.yaml

```diff
@@ -28,9 +28,9 @@ plugins:
     out: ./buf
     opt:
       - paths=source_relative
-  - remote: buf.build/grpc-ecosystem/plugins/openapiv2:v2.7.1-1
+  - plugin: buf.build/grpc-ecosystem/openapiv2:v2.15.2
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.

I've also bumped the versions in this PR to reference the latest available versions.